src/parser/jobs/mnk/index.tsx

```diff
@@ -1,6 +1,5 @@
 import {Trans} from '@lingui/macro'
 import React from 'react'
-import {Icon, Message} from 'semantic-ui-react'
```

Comment:
Sneaky! But gud acc

Reply:
This import isn't used anywhere, I'm surprised it doesn't trip the linter being left in there tbh.